docs/data-sources/route_control_profile.md

```diff
@@ -7,33 +10,66 @@ description: |-
   Data source for ACI Route Control Profile
 ---
 
-# aci_route_control_profile
+# aci_route_control_profile #
 
 Data source for ACI Route Control Profile
 
-## Example Usage
+## API Information ##
+
+* Class: [rtctrlProfile](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/rtctrlProfile/overview)
+
+* Supported in ACI versions: 1.0(1e) and later.
+
+* Distinguished Name Formats:
+  - `uni/tn-{name}/out-{name}/prof-{name}`
+  - `uni/tn-{name}/prof-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Tenants -> Networking -> L3Outs -> Route map for import and export route control`
+  - `Tenants -> Policies -> Protocol -> Route Maps for Route Control`
+
+## Example Usage ##
 
 ```hcl
-data "aci_route_control_profile" "check" {
-  parent_dn = aci_tenant.tenentcheck.id
-  name      = "one"
+
+data "aci_route_control_profile" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "test_name"
 }
 
-data "aci_route_control_profile" "check" {
+data "aci_route_control_profile" "example_l3_outside" {
   parent_dn = aci_l3_outside.example.id
-  name      = "route_control_profile_1"
+  name      = "test_name"
 }
+
 ```
 
-## Argument Reference
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - [aci_l3_outside](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/l3_outside) ([l3extOut](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extOut/overview))
+* `name` (name) - (string) The name of the Route Control Profile object.
+
+### Read-Only ###
 
-- `parent_dn` - (Required) Distinguished name of the parent object.
-- `name` - (Required) Name of router control profile object.
+* `id` - (string) The distinguished name (DN) of the Route Control Profile object.
+* `annotation` (annotation) - (string) The annotation of the Route Control Profile object.
+* `route_map_continue` (autoContinue) - (string) This action will be applied on all the entries which are part of per Peer BGP Route-Map. Only available on 'Tenants -> Policies -> Protocol -> Route Maps for Route Control'.
```

Comment:
Should use only applicable when parent_dn is set to tenant instead of providing the a UI link? 

Perhaps we should reconsider even mentioning this, because this is user responsibility of knowing the config they would like to push? Like if we start doing this, how do we handle UI version changes in our docs?

Reply:
Perhaps we should also change this action sentence to what the attribute does? So what auto-continue means.